Yubico.YubiKey/src/Yubico/YubiKey/YubiKeyFeature.cs

```diff
@@ -261,5 +261,10 @@ public enum YubiKeyFeature
         /// Allows temporarily disabling NFC until the next time the YubiKey is powered over USB.
         /// </summary>
         ManagementNfcRestricted,
+        
+        /// <summary>
+        /// Support for deleting PIV keys or moving PIV keys between slots.
+        /// </summary>
+        PivMoveOrDeleteKey
```

Comment:
Move this into the PIV section of features / enum values.

An application should never be persisting the values of the enumerations here, nor would it cross version boundaries - so the fact that the enum values are implicitly changing under the covers should be OK.

Reply:
Ah there were regions here, I saw them in the `YubikeyFeatureExtensions` but missed it here for some reason. Thanks!